src/utils/combineReducers.js

```diff
@@ -97,9 +98,16 @@ export default function combineReducers(reducers) {
     }
   });
 
+  if (sanityError) {
+    console.error(sanityError.message);
+  }
+
```

Comment:
The idea is to delay throwing the error but log as soon as possible. The console.error was suggested by @gaearon in the second comment of issue #717. I do agree that this is noisy, as the error is thrown shortly afterwards and you end up with the same error logged twice in succession.